types/tkatype/tkatype.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package tkatype defines types for working with the tka package.
```

Comment:
You may want to add a comment here not to add dependencies, because you want to avoid extra deps in the tka package.